src/bunit.web/TestDoubles/HttpClient/PlaceholderHttpClient.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Bunit.TestDoubles.HttpClient
+{
+	public class PlaceholderHttpClient : System.Net.Http.HttpClient
```

Comment:
Make all the placeholder classes internal.

```suggestion
	internal class PlaceholderHttpClient : System.Net.Http.HttpClient
```